Kernel/API/prctl_numbers.h

```diff
@@ -13,3 +13,5 @@
 #define PR_SET_COREDUMP_METADATA_VALUE 5
 #define PR_SET_PROCESS_NAME 6
 #define PR_GET_PROCESS_NAME 7
+#define PR_SET_NAME 8
+#define PR_GET_NAME 9
```

Comment:
We should probably not define new constants inside our kernel just to support porting over applications. Maybe these should be:

```
#define PR_SET_NAME PR_SET_PROCESS_NAME
#define PR_GET_NAME PR_GET_PROCESS_NAME
```

...so we reuse the existing constants, and this should go in `Userland/Libraries/LibC/sys/prctl.h` instead?

Reply:
I understand how I could define these in Userland, but that Kernel needs to know about them for the actual sys$prctl implementation.

Reply:
I could also patch zsh source to use the names we use (and dont exist in linux), that would need no changes to our API at all.

Reply:
> but that Kernel needs to know about them for the actual sys$prctl implementation.

Kernel already supports `PR_SET_PROCESS_NAME` (the value `6`). So that's why I suggested `#define PR_SET_NAME PR_SET_PROCESS_NAME` for userland.

Reply:
Alternate suggestion: rename our APIs to `PR_SET_NAME` and `PR_GET_NAME`. I don't see a reason to have our own unique names here.

Reply:
I think they are not absolutly identical in how the work if I read the docs correctly. Like, we are not truncating after 16 bytes at all. So that would be nothing I want to do in ports update PR I think.

Reply:
The Linux prctl sets thread name, rather than process name. Those are distinct strings in serenity, though they may be the same in Linux. It doesn't seem quite right to alias them... Though really anyone trying to set thread name should go through pthread_setname_np anyway.

Reply:
I decided to just not support the syscall in `zsh` for now. The benefit from it working for `jobs -Z` is just to small to invest the time that a clean implementation would need.